stroeer/curation/v1/core_curation_service.proto

```diff
@@ -4,7 +4,7 @@ syntax = "proto3";
  * @FileArticle ⚙︎ CurationService
  */
 
-package stroeer.curation.v1;
+package foo;
```

Comment:
?

Reply:
just to test the lint and breaking change detectin/gh integration. will be reverted before merging